mxfusion/components/variables/runtime_variable.py

```diff
@@ -97,3 +97,22 @@ def as_samples(F, array, num_samples):
         return array
     else:
         return F.broadcast_axis(array, axis=0, size=num_samples)
+
+
+def arrays_as_samples(F, arrays):
+    """
+    Broadcast the dimension of samples for a list of variables. If the number of samples of at least one of the variables is larger than one, all the variables in the list are broadcasted to have the same number of samples.
+
+    :param F: the execution mode of MXNet.
+    :type F: mxnet.ndarray or mxnet.symbol
+    :param arrays: a list of arrays considered to be broadcasted.
```

Comment:
```suggestion
    :param arrays: a list of arrays with samples to be broadcasted.
```

Maybe just call out explicitly that you assume the arrays here already have a dimension for samples, regardless of if they have more than 1 sample.